src/nvim/main.c

```diff
@@ -2115,3 +2093,27 @@ static void check_swap_exists_action(void)
     getout(1);
   handle_swap_exists(NULL);
 }
+
+/// Sets up the terminal window for use.
+///
+/// This must be done after resetting full_screen, otherwise it may move the
+/// cursor.
+///
+/// @remark We may call mch_exit() before calling this.
+static void term_init(void)
+{
+  Columns = 80;
+  Rows = 24;
```

Comment:
These are already the values initialized in globals.h so its not necessary :+1: 

I simply removed term_init and replaced by a call to `setbuf`(ui_flush is not required at this point)